Makefile

```diff
@@ -29,8 +31,8 @@ uninstall:
 	@rm -f "$(LIB_DIR)/systemd/system/grub-btrfs.path"
 	@rm -f "$(LIB_DIR)/initcpio/install/grub-btrfs-overlayfs"
 	@rm -f "$(LIB_DIR)/initcpio/hooks/grub-btrfs-overlayfs"
-	@# Arch Linux unlike distros only :
-	@if ! command -V mkinitcpio >/dev/null 2>&1; then \
+	@# Arch Linux UNlike distros only :
+	@if test "$(INITCPIO)" != true && test -d "$(LIB_DIR)/initcpio"; then \
```

Comment:
Interesting check for the directory - just thinking, what if we simply check if the directory is present, then install all initcpio stuff? That way we would not even need a parameter?

Reply:
Yeah makes sense. 😄 

Not even sure why I did that check for the mkinitcpio binary in the first place, rather than just for the directory...

Reply:
Ah hold on... That would also fail to install the hook in a chroot, wouldn't it ?

Reply:
Mmm true - you see, I also fell in this trap 😅 OK lets just go with the parameter 😁

Reply:
why is the condition reversed in uninstall by the way, shouldn't it also remove the files only if you say "make uninstall INITCPIO=true"?

Reply:
The old Makefile made it possible to install the hook on distros where it's useless. In such a case, the rogue folders/files should be removed.

On Arch, the directories are not empty, and should remain on the system. The hook itself is removed before the `@if` clause. (2 preceding commands)

Reply:
> shouldn't it also remove the files only if you say "make uninstall INITCPIO=true"?

Should the hook remain even after `grub-btrfs` is uninstalled ?  If so, we'd need an additional `if` for these 2 `rm` commands, I think.

Reply:
I guess no, hook is part of the package, so should not remain...